examples/internal/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -322,6 +322,38 @@ message UpdateV2Request {
 	google.protobuf.FieldMask update_mask = 2;
 }
 
+message Book {
+	// The resource name of the book.
+	//
+	// Format: `publishers/{publisher}/books/{book}`
+	//
+	// Example: `publishers/1257894000000000000/books/my-book`
+	string name = 1;
+
+	// Output only. The book's ID.
+	string id = 2;
+
+	// Output only. Creation time of the book.
+	google.protobuf.Timestamp create_time = 3;
+}
+
+message CreateBookRequest {
+	// The publisher in which to create the book.
+	//
+	// Format: `publishers/{publisher}`
+	//
+	// Example: `publishers/1257894000000000000`
+	string parent = 1;
+
+	// The book to create.
+	Book book = 2;
+
+	// The ID to use for the book.
+	//
+	// This must start with an alphanumeric character.
+	string book_id = 3;
+}
```

Comment:
Could you reference the AIP in a comment here to explain these two messages please?